policyengine_it/parameters/gov/inps/social_security/rates/executive.yaml

```diff
@@ -0,0 +1,21 @@
+description: Social security rates for executives
+brackets:
+  - threshold:
+      values:
```

Comment:
dont need values if we dont metadata:

just:
   threshold:
      000... 
    rate: